src/taco-cli/cli/run.ts

```diff
@@ -88,6 +91,12 @@ class Run extends commands.TacoCommandBase {
             if (!remoteConfig) {
                 throw errorHelper.get(TacoErrorCodes.CommandRemotePlatformNotKnown, platform);
             }
+            
+            // DeviceSync/LiveReload not compatible with remote
+            var deviceSync = commandData.options["livereload"] || commandData.options["devicesync"];
+            if (deviceSync) {
```

Comment:
Because, we can execute a remote platform in TACO without having to specify 'taco remote', It suffices to have a remote configuration set up. And when that's the case, the ```parseArgs``` code won't be sufficient to prevent us from running livereload on a remote target.

For example, on my Linux box, if I just issue: ```cordova run ios```, it runs it as remote platform.